packages/bottom-tabs/src/views/BottomTabBar.tsx

```diff
@@ -304,6 +304,9 @@ export function BottomTabBar({
 
   const tabBarBackgroundElement = tabBarBackground?.();
 
+  const { transform = [], ...restTabBarStyle } = (tabBarStyle ??
+    {}) as ViewStyle;
```

Comment:
Don't use `as` to ignore types. The type isn't `ViewStyle` but `StyleProp<ViewStyle>` - which can be a nested array and contain booleans and falsy values. Destructuring like this won't work in all scenarios.